### PR TITLE
refactored prWaitingRoom.html

### PR DESCRIPTION
**Pull request by briosolivares**

### CODE REVIEW

File: templates/prWaitingRoom.html [hunk 1]
```diff
@@ -21,11 +21,11 @@
     <script defer src="/__/firebase/init.js"></script> 
 
      <link href="https://unpkg.com/material-components-web@latest/dist/material-components-web.min.css" rel="stylesheet">
-    <link rel="stylesheet" type="text/css" href="prWaitingRoom.css">
-     <link rel="stylesheet" type="text/css" href="/temp/assets/css/bootstrap.min.css">
-    <link rel="stylesheet" type="text/css" href="/temp/assets/css/font-awesome.css">
-    <link rel="stylesheet" type="text/css" href="/temp/assets/css/templatemo-art-factory.css">
-    <link rel="stylesheet" type="text/css" href="/temp/assets/css/owl-carousel.css">
+    <link rel="stylesheet" type="text/css" href="../temp/assets/css/prWaitingRoom.css">
```
Comment (Mannendri) [inline templates/prWaitingRoom.html]: We've agreed to use the project directory as root for all hrefs. Change all of these hrefs (lines 24-28) to match this new naming system. You no longer need the escape sequence ".."
For example: 
<link rel="stylesheet" href="/public/assets/css/patientSignUp.css"/>

File: templates/prWaitingRoom.html [hunk 2]
```diff
@@ -154,23 +154,23 @@ <h2 class="mdc-dialog__title" id="my-dialog-title">Join room</h2>
     </div>
     <div class="mdc-dialog__scrim"></div>
 </div>
-<script src="app.js"></script>
+<script src="../FirebaseRTC/public/app.js"></script>
```
Comment (Mannendri) [inline templates/prWaitingRoom.html]: Similar to the previous comment, update this src to match the new naming system. The project directory is the root.

File: templates/prWaitingRoom.html [hunk 3]
```diff
@@ -154,23 +154,23 @@ <h2 class="mdc-dialog__title" id="my-dialog-title">Join room</h2>
...
-    <script src="/temp/assets/js/imgfix.min.js"></script> 
+    <script src="../temp/assets/js/owl-carousel.js"></script>
+    <script src="../temp/assets/js/scrollreveal.min.js"></script>
+    <script src="../temp/assets/js/waypoints.min.js"></script>
+    <script src="../temp/assets/js/jquery.counterup.min.js"></script>
+    <script src="../temp/assets/js/imgfix.min.js"></script> 
     
     <!-- Global Init -->
-    <script src="/temp/assets/js/custom.js"></script>
+    <script src="../temp/assets/js/custom.js"></script>
```
Comment (Mannendri) [inline templates/prWaitingRoom.html]: For all of the script tags above this comment, update the srcs to match the new naming system. The project directory is the root. Refer to the first comment on this file for an example.

**Review by lillianchou94 — APPROVED** [review]

**Comment by j0n3lson** [comment]
Fixed in #34

**Pull request closed without merging**